addon/templates/components/polaris-layout/annotation.hbs

```diff
@@ -1,14 +1,14 @@
 {{#polaris-text-container}}
   {{polaris-heading text=title}}
 
-  {{#if (or hasBlock description)}}
+  {{#if description}}
     <div data-test-annotation-description class="Polaris-Layout__AnnotationDescription">
-      {{#if hasBlock}}
-        {{yield}}
-      {{else}}
+      {{#if hasStringDescription}}
```

Comment:
do we need this? wouldn't `render-content` handle this correctly?

Reply:
We need to explicitly add the `p` wrapper if `description` is just a string